.github/workflows/tests.yml

```
@@ -0,0 +1,57 @@
+          echo "Prisma version: $(pnpm --filter cricketscoreapi prisma -v)"
+          NODE_ENV=test pnpm --filter cricketscoreapi test:coverage
+        env:
+          NODE_ENV: test
+          DATABASE_URL: ${{ secrets.DATABASE_URL }}
+          TEST_USER_TOKEN: ${{ secrets.TEST_USER_TOKEN }}
+          CLERK_PUBLISHABLE_KEY: ${{ secrets.CLERK_PUBLISHABLE_KEY }}
+          CLERK_SECRET_KEY: ${{ secrets.CLERK_SECRET_KEY }}
+          OPTIMIZE_API_KEY: ${{ secrets.OPTIMIZE_API_KEY }}
+
+      - name: "Upload Coverage"
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)